src/Hacks/aimbot.cpp

```diff
@@ -61,6 +62,8 @@ QAngle RCSLastPunch;
 
 int Aimbot::targetAimbot = -1;
 
+static xdo_t *xdo = xdo_new(NULL);
```

Comment:
Do we need to clean this up this after it allocates memory? 

Is there something like xdo_close()